src/Illuminate/Foundation/Console/Tinker/Presenters/EloquentCollectionPresenter.php

```diff
@@ -0,0 +1,31 @@
+<?php namespace Illuminate\Foundation\Console\Tinker\Presenters;
+
+use Psy\Presenter\ObjectPresenter;
+use Illuminate\Database\Eloquent\Collection;
+
+class EloquentCollectionPresenter extends ObjectPresenter {
+
+	/**
+	 * EloquentCollectionPresenter can present Collections.
+	 *
+	 * @param  mixed  $value
+	 * @return bool
+	 */
+	public function canPresent($value)
+	{
+		return $value instanceof Collection;
+	}
+
+	/**
+	 * Get an array of Collection object properties.
+	 *
+	 * @param  object  $value
+	 * @param  \ReflectionClass  $class
+	 * @param  int  $propertyFilter  One of \ReflectionProperty constants
```

Comment:
Laravel tends not to use parameter descriptions. Maybe you could describe this using a "long description".